docs/installation.md

```diff
@@ -0,0 +1,76 @@
+# Installation Guide
+
+To install and use AutoRA you need:
+
+- `Python` (version ">=3.8,<4") and
+- the `autora` package, including required dependencies specified in the `pyproject.toml` file.
+
+## Install `Python`
+
+You can install `Python`:
+
+- Using the instructions at [python.org](https://www.python.org), or
+- Using a package manager, e.g.
+  [homebrew](https://docs.brew.sh/Homebrew-and-Python), 
+  [pyenv](https://github.com/pyenv/pyenv),
+  [asdf](https://github.com/asdf-community/asdf-python), 
+  [rtx](https://github.com/jdxcode/rtx/blob/main/docs/python.md),
+  [winget](https://winstall.app/apps/Python.Python.3.8).
+
+If successful, you should be able to run python in your terminal emulator like this:
+```shell
+python
+```
+
+...and see some output like this:
+```
+Python 3.11.3 (main, Apr  7 2023, 20:13:31) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
+Type "help", "copyright", "credits" or "license" for more information.
+```
+## Install `autora`
+
+To install the PyPI `autora` package, run the following command:
+
+```shell
+pip install "autora"
```

Comment:
should we consistently use ```pip install "autora"``` or ```pip install autora```?

Reply:
good catch, thanks! on that point, i think we still have some cleaning up to do globally... eg, https://autoresearch.github.io/autora/workflow-tutorial/docs/interactive/Basic%20Usage/#:~:text=%23%20!pip%20install%20autora

i'll make an issue.